src/modules/chat_highlight_blacklist_keywords/index.js

```diff
@@ -184,6 +186,55 @@ class ChatHighlightBlacklistKeywordsModule {
             defaultValue: false,
             description: 'Automatically hide pinned highlights after 1 minute'
         });
+
+        settings.add({
+            id: 'soundHighlight',
+            name: 'Play Sound On Highlight',
+            defaultValue: false,
+            description: 'On highlight you hear a sound'
+        });
+
+        this.username = twitch.getChatServiceClient().configuration.username.toLowerCase();
+        this.soundHighlight = false;
+
+        this.setSoundHighlight = this.setSoundHighlight.bind(this);
+        this.handleSoundHighlight = this.handleSoundHighlight.bind(this);
+
+        this.setSoundHighlight(settings.get('soundHighlight'));
+        settings.on('changed.soundHighlight', this.setSoundHighlight);
+    }
+
+    setSoundHighlight(value) {
+        this.soundHighlight = value;
+    }
+
+    handleSoundHighlight(messageParts = []) {
+        const isMentioned = messageParts.some(({content}) => {
+            if (typeof content === 'object' && content !== null) {
+                const {recipient = '', currentUserMentionRelation} = content;
+
+                return recipient.toLowerCase() === this.username && typeof currentUserMentionRelation !== 'undefined';
+            }
+
+            return false;
+        });
+
+        if (!isMentioned) {
+            return;
+        }
+
+        let node = $(`#${CHAT_MENTION_NOTIFICATION_ID}`);
+
+        if (node.length) {
+            node[0].play();
+        } else {
+            node = document.createElement('audio');
```

Comment:
why do we need to make an audio element? can't we just load it in js and play it in js?